src/components/footer/_macro.njk

```diff
@@ -94,6 +94,10 @@
                         </div>
                     {% endif %}
 
+                    {% if params.newTabWarning is defined and params.newTabWarning %}
+                        <p class="u-fs-r footer__new-tab-warning">{{ params.newTabWarning }}</p>
```

Comment:
From the design, it looks like the lead in line should be `u-fs-s`
![image](https://user-images.githubusercontent.com/43346934/96899872-58556a80-1489-11eb-85bc-70cfa13659e9.png)